blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarterTest.java

```diff
@@ -0,0 +1,275 @@
+package com.rackspacecloud.blueflood.service;
+
+import static junit.framework.Assert.*;
+
+import org.junit.*;
+
+import java.io.IOException;
+
+public class BluefloodServiceStarterTest {
+
+    @Before
+    public void setUp() throws IOException {
+
+        Configuration config = Configuration.getInstance();
+
+        config.init();
+
+        // take from demo/local/config/blueflood.conf
+        config.setProperty(CoreConfig.CASSANDRA_HOSTS, "127.0.0.1:9160");
```

Comment:
I also think we should try to setup/instantiate configuration class, same as the way the main code does so that tests are close to the main code setup.